dali/python/backend_impl.cc

```diff
@@ -718,6 +718,11 @@ void ExposeTensor(py::module &m) {
       Returns a `TensorGPU` object being a copy of this `TensorCPU`.
       )code",
       py::return_value_policy::take_ownership)
+    .def("as_cpu", [](Tensor<CPUBackend> &t) -> Tensor<CPUBackend>& {
+          return t;
+        },
+      R"code(Bypass, since the object is already an instance of `TensorCPU`.)code",
```

Comment:
Is the `Bypass` a correct word here? Maybe `no-op` would be better? Up to you though.
```suggestion
      R"code(No-op, since the object is already an instance of `TensorCPU`.)code",
```

Reply:
If changing at all, I'd recommend `Passthrough` or `Returns self` over `No-op`, which is rather awkward for a pure function.